packages/fxa-payments-server/src/components/PaymentForm/index.tsx

```diff
@@ -154,7 +156,10 @@ export const PaymentForm = ({
       validator={validator}
       onSubmit={onSubmit}
       className="payment"
-      onChange={engage}
+      onChange={() => {
```

Comment:
We should try to avoid defining inline functions for event handlers like this. Better way might be to rename [the `engage()` callback](https://github.com/mozilla/fxa/blob/34b9bcccb5ecbc9fbc155bde2ae779962e466e3c/packages/fxa-payments-server/src/components/PaymentForm/index.tsx#L120) to a more generic `onChange` and move the `onChangeErrorDismiss()` call there.

(I've also just realized all the `onValidate` handlers are inline functions, which is poor form but I/we can refactor at some future point.)